apps/server/src/modules/deletion/uc/deletion-request.uc.ts

```diff
@@ -0,0 +1,216 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain';
+import { PseudonymService } from '@src/modules/pseudonym';
+import { UserService } from '@src/modules/user';
+import { TeamService } from '@src/modules/teams';
+import { ClassService } from '@src/modules/class';
+import { LessonService } from '@src/modules/lesson/service';
+import { CourseService } from '@src/modules/learnroom/service';
+import { CourseGroupService } from '@src/modules/learnroom/service/coursegroup.service';
+import { FilesService } from '@src/modules/files/service';
+import { AccountService } from '@src/modules/account/services/account.service';
```

Comment:
Please replace all @src/modules/* to @modules/*
Please import services form @modules/account maybe you need create the service to 
~/modules/account/index.ts

Reply:
done